packages/flutter/test/widgets/scrollable_animations_test.dart

```diff
@@ -80,6 +62,37 @@ void main() {
 
     expect(SchedulerBinding.instance.transientCallbackCount, equals(1), reason: 'Expected an animation.');
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/14452
```

Comment:
I am not really not familiar with the rule of regression test, but do we need this comment given that this is not a regression issue?

Reply:
I've noticed this trend showing up in recent PRs and was wondering what you and @goderbauer thought. It's not in the style guide, but some changes have included this notation.
Example: #37187

Reply:
I have been asked to add this comment when it is a regression issue. But I fine with either

Reply:
Adding this link is good practice. It gives people that look at it in the future context.